cli/internal/commands/authorize_cluster/generator.go

```diff
@@ -263,6 +263,24 @@ func printHelmValues(obj interface{}, w io.Writer) error {
 		},
 	}
 
+	// Remove values we already included
+	delete(secret.Data, "STORMFORGE_AUTHORIZATION_CLIENT_ID")
+	delete(secret.Data, "STORMFORGE_AUTHORIZATION_CLIENT_SECRET")
+
+	// The Helm chart hard codes these to the production default values
```

Comment:
Could we put this behind a (hidden?) flag? Thinking more about internal use case

Reply:
err I guess my comment is more around including it in `vals` above